### ISSUE TITLE
Highlights whole page?

### ISSUE THREAD

**Issue by IanRogers** [kazmiya/dokuwiki-plugin-editsections2#1]
I'm using this plugin with Anteater.  The buttons are placed perfectly (thank you! :-)  but when you mouse-over a button it seem the whole page is highlighted and not just the section.

Are other people seeing this? If not I'd guess it's an interaction with another plugin...

**Comment by kazmiya** [comment]
I've checked with a plain DokuWiki installation, but couldn't confirm such a problem.

First of all, make sure you're not using old js cache. Please refer to the url below to purge your old js cache.

  http://www.dokuwiki.org/devel:caching#purging_the_cache

If you still have problem after purging js cache, I guess any of your installed plugins conflict with the section highlighting function. If so, could you help finding a plugin which causes this problem? I think you can find that plugin if you turn off your installed plugins one by one.

If you find the cause, please let me know the name of the plugin. I'll make efforts to solve the problem.

**Comment by samh** [comment]
I'm having the same issue, on 2010-11-07a "Anteater", though other than the highlighting it seems to work correctly. The other editsections plugin wasn't working for me at all, since I upgraded to this version. I tried a fresh installation, and both work, so I'm trying to work out what's different in my main installation. The only other plugin I have enabled is the popularity plugin.

**Comment by kazmiya** [comment]
@samh
Hmm... I'm using with popularity plugin but no problem occurs.

I think the problem could be solved if you purge your old js cache. The old js cache problem is likely to occur if you installed the plugin manually (i.e. not via plugin manager). Purging old js cache doesn't work?

**Comment by samh** [comment]
Right, the popularity plugin is also on the fresh install that works.

I installed through the plugin manager (using the master branch tarball).

I tried purging the js cache. "lib/exe/js.php?purge=true" showed some javascript in the browser; I'm guessing that's correct. Also manually removed everything in the cache directory and cleared my browser cache, so I'm pretty sure it's something about my configuration. It's been through several upgrades, and I use rewrites and have several directories outside the document root (as described on [security](http://www.dokuwiki.org/security)). I'll probably end up rebuilding the installation from a fresh install, and hopefully see which change (if any) breaks it. I'll post here if I find what it was.

**Comment by samh** [comment]
Well, it worked when I moved everything back under the document root, so either something about that was messing up the plugins or I didn't do something right when I configured it. Maybe it's time to just rely on Apache to protect those directories, making everything simpler.

**Comment by samh** [comment]
I spoke too soon. It seems to have been something about the process of copying the files to another directory that fixed it. It seems 'cp -a' wasn't able to copy all the permissions (I don't have root access on my web host), and I had to fix some afterward to get everything else working. Tentatively everything looks okay, but it's really bugging me that I couldn't find exactly what caused the issue.
